feature/gnoi/system/tests/complete_chassis_reboot/complete_chassis_reboot_test.go

```diff
@@ -168,6 +173,25 @@ func TestChassisReboot(t *testing.T) {
 				t.Errorf("Get boot time: got %v, want > %v", bootTimeAfterReboot, bootTimeBeforeReboot)
 			}
 
+			startComp := time.Now()
+			t.Logf("Wait for all the components on DUT to come up")
+
+			for {
```

Comment:
It could be infinite loop if the first if condition is always false. 

Errorf can be replaced with Fatalf to avoid the infinite loop.

Reply:
@sezhang2   Thanks for the review. Errorf replaced with Fatalf. Please check.